virt/install/installing-virt-cli.adoc

```diff
@@ -8,6 +8,11 @@ Install {VirtProductName} to add virtualization functionality to your {product-t
 cluster. You can subscribe to and deploy the {VirtProductName} Operators by
 using the command line to apply manifests to your cluster.
 
+[NOTE]
+====
+To specify the nodes where you want {VirtProductName} to install its components, xref:../../virt/install/virt-specifying-nodes-for-virtualization-components.adoc#virt-specifying-nodes-for-virtualization-components[configure node placement rules in the HyperConverged Cluster CR].
+====
+
```

Comment:
There's a "Prerequisites" heading right below here that should get an anchor/ID added.